fiware-region-sanity-tests/tests/fiware_region_base_tests.py

```diff
@@ -25,146 +25,137 @@
         self.assertNotEqual(len(image_list), 0, "Image list is empty")
+        self.logger.debug("Found %d available images", len(image_list))
 
-    def test_there_are_init_images(self):
+    def test_cloud_init_aware_images(self):
         """
-        Test 03: Check if the Region has images with 'init' in the name.
+        Test whether region has 'cloud-init-aware' images (suitable for blueprints)
         """
         image_list = self.nova_operations.get_image_list()
-        self.assertIsNotNone(image_list, "Image list is empty")
-        self.assertNotEqual(len(image_list), 0, "Image list is empty")
 
-        found = False
-        for image in image_list:
-            if 'init' in image['name']:
-                found = True
-                break
-        self.assertTrue(found, "No 'init' images has been found")
+        # Filter out images with 'init' in its name
+        image_list = [image for image in image_list if 'init' in image.name]
+
+        # Filter out images with 'sdc_aware' metadata attribute set to True
+        image_list = [image for image in image_list if image.metadata.get('sdc_aware', "false").lower() == "true"]
+
+        self.assertNotEqual(len(image_list), 0, "Cloud-init-aware image list is empty")
+        self.logger.debug("Found %d images", len(image_list))
 
     def test_base_image_for_testing_exists(self):
         """
-        Test 04: Check if the Region has the BASE_IMAGE_NAME used for testing
+        Test whether region has the image used for testing
         """
-        image_id = self.nova_operations.find_image_id_by_name(image_name=BASE_IMAGE_NAME)
-        self.assertIsNotNone(image_id, "Problems retrieving the image '{}'".format(BASE_IMAGE_NAME))
+        image_id = self.nova_operations.find_image_id_by_name(BASE_IMAGE_NAME)
+        self.assertIsNotNone(image_id, "Problems retrieving image '%s'" % BASE_IMAGE_NAME)
 
     def test_create_security_group_and_rules(self):
         """
-        Test 05: Check if it is possible to create a new Security Group with rules.
+        Test creation of a new security group with rules
         """
-        sec_group_name = "testing_sec_group"
-        sec_group_id = self.nova_operations.create_security_group_and_rules(sec_group_name)
-        self.assertIsNotNone(sec_group_id, "Problems creating Security Group")
 
-        self.test_world['sec_groups'].append(sec_group_id)
+        # skip if not all security groups were deleted
+        if self.test_world['sec_groups']:
+            self.skipTest("Not all the security groups were deleted")
+
+        sec_group_name = TEST_SEC_GROUP_PREFIX
+        try:
+            sec_group_id = self.nova_operations.create_security_group_and_rules(sec_group_name)
+            self.assertIsNotNone(sec_group_id, "Problems creating security group '%s'" % sec_group_name)
+            self.test_world['sec_groups'].append(sec_group_id)
+        except Forbidden as e:
```

Comment:
Are you sure that the exception forbidden is related to the message "Quota exceeded when creating a ..."?